panic: index out of range in InteractiveMultiselectPrinter

When there are no search matches when the user presses enter, the `InteractiveMultiselectPrinter` panics:

```
Please select your options: c
enter: select | tab: confirm | left: none | right: all | type to filter

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/pterm/pterm.(*InteractiveMultiselectPrinter).Show.func1({0xd, {0x0, 0x0, 0x0}, 0x0})
	/home/adrian/go/pkg/mod/github.com/pterm/pterm@v0.12.45/interactive_multiselect_printer.go:231 +0x1032
atomicgo.dev/keyboard.Listen(0xc00010aab0)
	/home/adrian/go/pkg/mod/atomicgo.dev/keyboard@v0.2.8/keyboard.go:107 +0x19b
github.com/pterm/pterm.(*InteractiveMultiselectPrinter).Show(0xc000190000, {0x0, 0x0, 0xc0000061a0?})
	/home/adrian/go/pkg/mod/github.com/pterm/pterm@v0.12.45/interactive_multiselect_printer.go:118 +0x71e
```

--- a/interactive_multiselect_printer.go
+++ b/interactive_multiselect_printer.go
@@ -227,8 +227,10 @@ func (p *InteractiveMultiselectPrinter) Show(text ...string) ([]string, error) {
 			cancel()
 			return true, nil
 		case keys.Enter:
-			// Select option if not already selected
-			p.selectOption(p.fuzzySearchMatches[p.selectedOption])
+			if len(p.fuzzySearchMatches) > 0 {
+				// Select option if not already selected
+				p.selectOption(p.fuzzySearchMatches[p.selectedOption])
+			}
 			area.Update(p.renderSelectMenu())
 		}
 

